nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/PropertyCommentManager.java

```diff
@@ -203,8 +200,9 @@ public Comment createComment(CoreSession session, Comment comment)
         }
 
         return CoreInstance.doPrivileged(session, s -> {
-            String path = getCommentContainerPath(s, parentId);
-            DocumentModel commentModel = s.createDocumentModel(path, COMMENT_NAME, COMMENT_DOC_TYPE);
+            DocumentModel documentToComment = s.getDocument(ancestorRef);
```

Comment:
`documentToComment` can lead to miscomprehension, for instance: for a reply what is the document to comment? The document whose we try to reply to or the top level document?